NavigationReactNative/src/ios/NVSceneController.m

```diff
@@ -49,6 +49,16 @@ - (void)viewWillAppear:(BOOL)animated
 
     [navigationBar updateColors];
 
+    NSInteger crumb = [self.navigationController.viewControllers indexOfObject:self];
+    UIViewController *previousViewController = crumb > 0 ? [self.navigationController.viewControllers objectAtIndex:crumb - 1] : nil;
```

Comment:
Change from `previousViewController` to `previousController` to match with the code `NVNavigationBarView`